prediction_market_agent/agents/arbitrage_agent/deploy.py

```diff
@@ -103,7 +103,7 @@ def get_correlated_markets(self, market: AgentMarket) -> list[CorrelatedMarketPa
             limit=self.max_related_markets_per_market,
             text=market.question,
             filter_on_metadata={
-                "close_time_timestamp": {"gte": utcnow().timestamp() + 3600}
+                "close_time_timestamp": {"$gte": int(utcnow().timestamp() + 3600)}
```

Comment:
How come it worked before? 🤔

Reply:
Btw not needed to implement in this PR, just recommendation readibility purposes one can do `(utcnow() + timedelta(days=1)).timestamp()`, then comment like `# closing 1h from now` isn't needed and one can always easily change for different timedeltas without calculating seconds.